x-pack/plugins/lens/public/indexpattern_datasource/help_popover.scss

```diff
@@ -6,6 +6,7 @@
   @include euiYScrollWithShadows;
   max-height: 40vh;
   padding: $euiSizeM;
+  margin: -$euiSizeM;
```

Comment:
As per my previous comment, you can do away with the negative margining CSS if you go the route of applying a `paddingSize` prop to the `EuiPopoverTitle` component.